opsim/Proposal_single_filter_northern_ecliptic_spur.md

```diff
@@ -0,0 +1,17 @@
+# Northern Ecliptic Spur In a Single Filter
+
+A Proposed OpSim Experiment.
+
+*Proposer(s): Meg Schwamb*
+
+*Issue Thread: NA*
+
+
+## Proposed Changes to the Observing Strategy
+
+Observe the Northern Ecliptic Spur region in a single filter (either r or g) only
```

Comment:
Might be worth spelling out how this relates to the usual Northern Spur proposal (specifying which simulation - minion1016? - you are proposing a variation of.  It sounds as though you are suggesting to reduce the overall number of visits in this region, but it'd be good to be specific.

Reply:
There are improvements coming to opsim and I presume new simulations, so I'm not sure it's worth specifying the specific flavor.  I need to know how any changes to the North Ecliptic Spur will impact Solar System science especially once rolling cadence is implemented. So I'm saying to only observe the northern ecliptic spur fields in  a single filter (r or g).

Reply:
OK cool - when @ivezic and his team circle round to your proposal, they'll ask for more detail about how you want the simulation set up. Until then, I'll merge this in.